2020-03-Autumn/1_pointer_techniques/README.md

```diff
@@ -105,87 +117,46 @@ Go to [Solution](#s3)   [Top](#top)
 <!-- Don't remove -->
 <a name="s1"/>
 
-### 1. Max Consecutive Ones
+### 1. Remove Duplicates from Sorted Array
 
-Source: [LeetCode](https://leetcode.com/problems/max-consecutive-ones/)
+[LeetCode](https://leetcode.com/problems/remove-duplicates-from-sorted-array/)
 
-#### Brute Force Solution
+#### Optimal solution
 
-The direct solution would go through the array, find the first 1, and 
```

Comment:
Add a note here about what technique you're using here (i.e, Fast and slow pointers) so it's extra clear